FinniversKit/Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepContentView/TransactionStepContentViewModel.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import Foundation
+
+public protocol TransactionStepContentViewModel {
+    var title: String? { get }
+    var body: NSAttributedString? { get }
+    /*
```

Comment:
Will change this

Reply:
Introducing a new property `nativeBody` in addition to `body`. The difference is that `nativeBody` will only be provided by the backend if `body` contains the `<a href>` tag.